ChatSecure/Classes/Controllers/OTRDatabaseManager.m

```diff
@@ -130,7 +130,13 @@ - (BOOL)setupYapDatabaseWithName:(NSString *)name
     
     self.database = [[YapDatabase alloc] initWithPath:databasePath
                                            serializer:nil
-                                         deserializer:nil
+                                         deserializer:^ id (NSString *collection, NSString *key, NSData *data){
+                                             if ([collection isEqualToString:@"OTRXMPPPresenceSubscriptionRequest"]) {
```

Comment:
I'm not sure if this is what we should be doing here. I think we should keep the `OTRXMPPPresenceSubscriptionRequest` object around (stubbed out) so the deserialization operation doesn't fail, but then do a better job of making sure that we aren't doing things that would load objects from that collection in the first place.

We did something similar for the `OTRMessage` vs `OTRBaseMessage` migration.